### PR TITLE
Restructure ES6

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the Awesome Books website was restructured to use ES6 modules and syntax. The previous Awesome Books site was rebuilt this way, but still with the original functionality.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  document.getElementById('date').innerHTML = dt;
+
+  document.getElementById('addbtn').addEventListener('click', () => {
+    a.addBooks();
+  });
+
+  // Remove button event listener
+  document.addEventListener('click', (event) => {
+    if (event.target.classList.contains('remove-btn')) {
+      // eslint-disable-next-line radix
```
Comment (AbayomiOlaoye) [inline index.js]: - [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (AbayomiOlaoye) [inline modules/books.js]: - [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

File: modules/books.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  saveToLocalStorage() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  // Add event listeners to remove buttons
+  addRemoveButtonListeners() {
+    const removeButtons = document.getElementsByClassName('remove-btn');
+    for (let i = 0; i < removeButtons.length; i += 1) {
+      const button = removeButtons[i];
+      // eslint-disable-next-line prefer-destructuring
```
Comment (AbayomiOlaoye) [inline modules/books.js]: - [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

File: modules/books.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      const bookId = button.dataset.bookId;
+      button.addEventListener('click', () => {
+        // eslint-disable-next-line radix
```
Comment (AbayomiOlaoye) [inline modules/books.js]: - [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

File: modules/navigation.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (AbayomiOlaoye) [inline modules/navigation.js]: - [ ] Kindly bear in mind that the use of a linter is a standard practice that ensures commitment to best practices in the real world in order to ensure ease of understandability, readability, maintainability, and scalability of code. In light of this, kindly resolve this linting issue without disabling it with this line :pray:

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @dvcorreia1984 :wave: :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- App functions as expected :heavy_check_mark: 
- Good use of ES6 syntax (modules, let, const, and arrow functions) :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (dvcorreia1984) [reply index.js]: Thank you Abayomi, I gave attention to these problems!

**Review by topeogunleye — APPROVED** [review]
### Hi @dvcorreia1984, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed eslint disablers. ✔️ 
 
## [Optional] suggestions:
👉 One optional suggestion for improvement is to follow the convention for naming your branch. Usually, branch names should use the past tense verb to indicate that the feature has been implemented. For example, instead of `Restructure-ES6`, you could use `Restructured-ES6`. This would make your branch name more consistent and clear for future pull requests.

- [ ]  Next time, and for future commits, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**